ui/src/kapacitor/components/RuleMessage.js

```diff
@@ -20,16 +40,35 @@ export const RuleMessage = React.createClass({
     enabledAlerts: arrayOf(string.isRequired).isRequired,
   },
 
+  getInitialState() {
+    return {
+      selectedAlert: null,
+      selectedAlertProperty: null,
+    };
+  },
+
+  handleChangeMessage() {
+    const {actions, rule} = this.props;
+    actions.updateMessage(rule.id, this.message.value);
+  },
+
   handleChooseAlert(item) {
     const {actions} = this.props;
     actions.updateAlerts(item.ruleID, [item.text]);
+    actions.updateAlertNodes(item.ruleID, item.text, '');
+    this.setState({selectedAlert: item.text});
   },
 
   render() {
-    const {rule, actions} = this.props;
-    const alerts = this.props.enabledAlerts.map((text) => {
+    const {rule, actions, enabledAlerts} = this.props;
+    const defaultAlertEndpoints = DEFAULT_ALERTS.map((text) => {
       return {text, ruleID: rule.id};
     });
+
+    const alerts = enabledAlerts.map((text) => {
+      return {text, ruleID: rule.id};
+    }).concat(defaultAlertEndpoints);
```

Comment:
There's an issue here with SMTP.  SMTP can be both preconfigured and default.  So, we run the risk of it showing up twice.